WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift

```diff
@@ -48,6 +69,12 @@ import WordPressShared
     private var titleLabelMaxWidthConstraint: NSLayoutConstraint?
     private var titleLabelTopConstraint: NSLayoutConstraint?
 
+    //For No results on connection issue
+    private var reachability = Reachability.forInternetConnection()
```

Comment:
It looks like we have no intention of mutating `reachability` so we can probably use `private let` for this.